biahub/deconvolve.py

```diff
@@ -22,7 +24,46 @@
 from biahub.cli.utils import estimate_resources, get_output_paths, yaml_to_model
 
 
-@click.command()
+def compute_tranfser_function(
```

Comment:
@talonchandler @ieivanov Is there a call in waveorder to compute the transfer function? If yes, it is best to use library call.

Reply:
If there is a need for this method - please add a comment.

Reply:
I 100% agree that there's some consolidation that can happen here, and I have this top of mind (especially because I'm reconstructing mantis and other datasets lately). 

I propose we complete this refactor, complete the waveorder-recorder merge, then I will lead a reconstruction refactor.   

IMO no need to block this PR on this point.

Reply:
That sounds good, @talonchandler. Please start an issue to remind yourself to fix this later.

Reply:
[New issue here](https://github.com/czbiohub-sf/biahub/issues/63). Resolving this subdiscussion.